File: Beecrowd/Beginner./1064.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
    double num, sum = 0;
    int count = 0;

    for (int i = 0; i < 6; i++) {
        cin >> num;
        if (num > 0) {
            sum += num;
            count++;
        }
    }

    cout << count << " valores positivos" << endl;
    cout << fixed << setprecision(1) << sum / count << endl;

    return 0;
}

File: Beecrowd/1. Beginner/1064/1064.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        double number, sum = 0;
        int count = 0;

        for(int i=0; i<6; i++) {
            number = input.nextDouble();
            if(number > 0) {
                sum += number;
                count++;
            }
        }

        System.out.println(count + " valores positivos");
        System.out.printf("%.1f", sum/count);

        input.close();
    }
}
